eth/watchers/stub.go

```diff
@@ -35,3 +39,37 @@ func newStubUnbondLog() types.Log {
 
 	return log
 }
+
+func newStubNewRoundLog() types.Log {
```

Comment:
Can we structure this function in a similar manner as the other stub log methods? i.e. Use hex strings and add comments for what the non-event signature topics + the log data decodes to

Reply:
I'm not using real-world logs but arbitrary values, in which case it seemed less tedious and more robust to encode the topics and data from scratch. 

What I can do is not hardcode the roundnumber as a byte slice but do something like

`common.LeftPadBytes(big.NewInt(8).Bytes(), 32)`

Or I could figure out the hex string representations first, it's as you wish.